srtcore/core.cpp

```diff
@@ -11930,7 +11934,7 @@ void CUDTGroup::syncWithSocket(const CUDT& core)
     // [[using locked(m_GroupLock)]];
 
     set_currentSchedSequence(core.ISN());
-    setInitialRxSequence(core.m_iPeerISN);
+    setInitialRxSequence(); // XXX NOT core.m_iPeerISN
```

Comment:
XXX stands for FIXME?

Reply:
There's nothing to "fix me", this is only to catch an eye. Unlikely that his is wrong, but this parameter has always been ignored. This is for further investigation, why this parameter was passed.